ios/Toodle/Extensions/Date+Int64.swift

```diff
@@ -6,6 +6,6 @@ import Foundation
 
 extension Date {
     func asInt64Timestamp() -> Int64 {
-        return Int64(self.timeIntervalSince1970)
+        return Int64(self.timeIntervalSince1970 * 1000000)
```

Comment:
If this is supposed to be a millisecond timestamp:

```
self.timeIntervalSince1970 * 1_000
```

If microseconds, use `1_000_000` and call this `toMicroseconds`.